cpp/ql/src/semmle/code/cpp/Element.qll

```diff
@@ -2,13 +2,31 @@ import semmle.code.cpp.Location
 private import semmle.code.cpp.Enclosing
 private import semmle.code.cpp.internal.Type
 
+cached @element resolveElement(@element e) {
+  if isClass(e)
+  then result = resolve(e)
+  else result = e
+}
```

Comment:
If this predicate is only for use in this file, shouldn't we make it private?

Reply:
Done